### PR TITLE
Edited the syntax and examples for border-left.md

### LINKED ISSUE #3641: [Edit] CSS [Edit] CSS Borders: border-left
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

[PLEASE DO NOT WORK ON THIS ISSUE UNTIL YOU HAVE BEEN ASSIGNED] We would like an edit to a term within CSS Borders. The term entry lives in docs/content/css/concepts/borders/terms/border-left/border-left.md

Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

Please:
1) Add supplemental media (preferred .png or .gif, <1MB).
2) Media (images, gifs) should accurately represent the existing code blocks in the example sections and fit in-line. ex: [See last image here](https://www.codecademy.com/resources/docs/html/attributes/title)

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by SanchitMahajan236**
### Description

Closes #3641 
I edited border-left.md as per the issue assigned to me. I added more description of the syntax and make the "note:" line a blockquote as per the style guide, and also an additional example reflecting how border-right does not require 3 parameters and will assign default values.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/css/concepts/borders/terms/border-left/border-left.md [hunk 1]
```diff
@@ -22,30 +22,53 @@ Shorthand property that defines the width, color, and style of the left border o
...
 Set the left border of the `h1` element to be green, 3 pixels thick, and dotted:
 
 ```css
 h1 {
+  width: fit-content;
   border-left: green 3px dotted;
 }
 ```
 
+![Example 1](https://github.com/SanchitMahajan236/Codeacademy_docs/assets/116559051/2fe25faa-9e11-4095-a0e7-cb71ce6a7a39)
```
Comment (SSwiniarski) [inline content/css/concepts/borders/terms/border-left/border-left.md:47]: You need to follow the instructions for file hosting as described in the style guide [here](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md).

File: content/css/concepts/borders/terms/border-left/border-left.md [hunk 2]
```diff
@@ -22,30 +22,53 @@ Shorthand property that defines the width, color, and style of the left border o
...
 
 ```css
 h1 {
-  border-left: 4px dashed blue;
+  width: fit-content;
+  border-left: 3px dashed #0000ff;
 }
 ```
+
+![Example 2](https://github.com/SanchitMahajan236/Codeacademy_docs/assets/116559051/fcd95df2-78c2-4f57-9ca1-7b42334110cb)
```
Comment (SSwiniarski) [inline content/css/concepts/borders/terms/border-left/border-left.md:60]: You need to follow the instructions for file hosting as described in the style guide [here](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md).

File: content/css/concepts/borders/terms/border-left/border-left.md [hunk 3]
```diff
@@ -22,30 +22,53 @@ Shorthand property that defines the width, color, and style of the left border o
...
+not specified, it will default to `medium`.
+
+```css
+h1 {
+  width: fit-content;
+  border-left: rgb(0, 0, 0) solid;
+}
+```
+
+![Example 3](https://github.com/SanchitMahajan236/Codeacademy_docs/assets/116559051/9f4a99e3-9ecc-45f0-bb64-153c3fa9a611)
```
Comment (SSwiniarski) [inline content/css/concepts/borders/terms/border-left/border-left.md:74]: You need to follow the instructions for file hosting as described in the style guide [here](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md).

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@SanchitMahajan236 I did a first review and you need to upload the images and change the links in line with our You need to follow the instructions for file hosting as described in the style guide [here](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md)..

**Review by SSwiniarski — APPROVED** [review]
@SanchitMahajan236 Looks good for a second review.

**Review by caupolicandiaz — APPROVED** [review]
@SanchitMahajan236 @SSwiniarski LGTM

**Comment by caupolicandiaz** [comment]
@SanchitMahajan236 The CLA needs to be updated

**Comment by SanchitMahajan236** [comment]
> @SanchitMahajan236 The CLA needs to be updated

@caupolicandiaz Done

**Pull request merged**